MonoGame.Framework/Platform/Input/GamePad.iOS.cs

```diff
@@ -59,7 +59,7 @@ private static GamePadCapabilities GetCapabilities(GCController controller)
             //All iOS controllers have these basics
             var capabilities = new GamePadCapabilities()
             {
-                IsConnected = false,
+                IsConnected = true,
```

Comment:
I think this should default to false, because right now even after the controller is turned off, it still says connected.

Reply:
if the code reach this point the game controller is connected otherwise wouldn't be listed in `GCController.Controllers`. Also we never update capabilities `IsConnected` to `true` only the state.
if the game controller is not connected `PlatformGetCapabilities` will return `GamePadCapabilities { IsConnected = false };`